coins/src/adapters/tokenMapping.json

```diff
@@ -6160,6 +6160,11 @@
       "to": "coingecko#ondo-us-dollar-yield",
       "decimals": 18,
       "symbol": "ausdy"
+    },
+    "uusdn": {
```

Comment:
this should be a defillama chain slug

Reply:
The other tokens above are also from Noble and don't have a slug.